lightly/active_learning/scorers/detection.py

```diff
@@ -65,24 +66,79 @@ def _prediction_margin(model_output: List[ObjectDetectionOutput]):
             # set the score to 0 if there was no bounding box detected
             score = 0.
         scores.append(score)
-    return np.asarray(scores)
+
+    return np.asarray(scores), "objectness_least_confidence"
```

Comment:
Why not return the enum generated by the swagger codegen?

Reply:
Because the new names are not implemented yet on the api side, but will be created with https://github.com/lightly-ai/lightly-core/issues/495
My proposal is the following:
- merge this PR
- do https://github.com/lightly-ai/lightly-core/issues/495
- do https://github.com/lightly-ai/lightly/issues/330
Then the E2E tests work again and this time for all scores and not only `prediction-margin`

Reply:
Ah, so you had the issues for changing the openapi specs, good.